docs/docs/deploying-to-heroku.md

```diff
@@ -8,7 +8,7 @@ Set the `heroku/node.js` and `heroku-buildpack-static` buildpacks on your applic
 
 app.json:
 
-```
+```json
```

Comment:
You could remove the `app.json` above this code snippet and use it as a title for the code block:

`json:title=app.json`